workshop_sections/word2vec/README.md

```diff
@@ -1,120 +1,178 @@
 
-## Training the model
+After you have completed this, set `MY_BUCKET=gs://my-bucket` with your name, for the commands below.
 
-To define your model, run:
+Then submit a training job with:
 
 ```
-import tensorflow as tf
-import os
-import time
-workshop_root = os.path.dirname(os.path.dirname(os.getcwd()))
-output_dir = os.path.join(workshop_root, 'workshop-data', 'output', str(int(time.time())))
-tf.logging.set_verbosity(tf.logging.INFO)
-word2vec_model = tf.contrib.learn.Estimator(model_fn=word2vec.make_model_fn(index, counts.tolist()), model_dir=output_dir)
+gcloud beta ml jobs submit training myword2vectraining \
+  --staging-bucket $MY_BUCKET \
+  --module-name word2vec.task \
+  --package-path word2vec \
+  --config config.yaml \
+  -- \
+  --output-path $MY_BUCKET/word2vec/output/run1 \
+  --vocab-file gs://tf-ml-workshop/word2vec/data/text8-vocab.tsv \
+  --train-data-file gs://tf-ml-workshop/word2vec/data/text8-train.pb2 \
+  --eval-data-file gs://tf-ml-workshop/word2vec/data/text8-eval.pb2
 ```
 
-To train with the in memory numpy arrays, run:
+This will output TensorBoard summaries, and model checkpoints to `$MY_BUCKET/word2vec/output/run1`
 
-```
-word2vec_model.fit(x=targets, y=contexts, batch_size=256, max_steps=int(len(targets)/256))
-```
+Additional configuration can be added in the `config.yaml` file. For the schema of this file check out the [Google Cloud ML Docs](https://cloud.google.com/ml/reference/configuration-data-structures#yaml_configuration_file)
+
+## Viewing Embeddings With TensorBoard
+
+To view embeddings (and other summaries) with TensorBoard, simply start TensorBoard pointing it at your output directory (in the Cloud or locally). For example:
 
-To train with the pre-preprocessed TFRecords, run:
 ```
-word2vec_model.fit(input_fn=word2vec.input_from_files(word2vec.GCS_SKIPGRAMS, 256))
+tensorboard --logdir $MY_BUCKET/word2vec/output/run1
 ```
 
-In another shell (or outside your docker container), run
+## Loading Your Model For Prediction
 
-```sh
-tensorboard --logdir workshop-data/output
-```
+This word2vec model predicts similarities between words using it's trained embeddings.
 
-to watch the training outputs.
+When given a vector of words, the word2vec model outputs a generator of prediction tensors, and defers computation until `next()` is called on the generator.
 
-## Using the model to predict
+For this model, the generator will output a dictionary of two tensors: 'predictions'` are the words most similar to the provided words, while `'values'` is a measure of their similarity to the provided words.
 
-This word2vec model predicts similarities between words using it's trained embeddings.
+Since we defined our model with an `Estimator` we can load the trained model into a python intepreter fairly easy:
```

Comment:
Hmmm it's pretty short, but no reason we couldn't... except wouldn't we have to put model (and all it's dependencies?) in the notebook? How does importing from non-notebooks work?